services/app/src/components/views/quest/cardtemplates/combat/Victory.tsx

```diff
@@ -20,63 +22,103 @@ export interface DispatchProps {
 
 export interface Props extends StateProps, DispatchProps {}
 
-export default function victory(props: Props): JSX.Element {
-  const contents: JSX.Element[] = [];
-  const theHorror = (props.settings.contentSets.horror === true);
+function renderHealing(props: Props): JSX.Element|null {
+  if (props.victoryParameters.heal && props.victoryParameters.heal > 0 && props.victoryParameters.heal < MAX_ADVENTURER_HEALTH) {
+    return (
+      <p>All adventurers <strong>regain {props.victoryParameters.heal} health</strong> (even if at 0 health).</p>
```

Comment:
Keeping with the bolded first word style, we could maybe change this to **Heal** X health (even if at 0 health)? Or something along those lines